src/components/OrganizationListGrid.tsx

```diff
@@ -26,7 +26,7 @@ export const OrganizationListGrid = ({
 			>
 				<ListGridItemTitle>{organization.name}</ListGridItemTitle>
 				<ListGridItemDetails>
-					{organization.employerIdentificationNumber}
+					{organization.taxId}
```

Comment:
Prettier is tripping on this because with the shorter attribute name, it no longer needs to be on its own line. Can you run Prettier on this and force-push again?

Reply:
Argh sorry about that -- this is what I get for trying to sneak in fixes after hours 😅

Reply:
No it's fine, I realize what I should have done (since CI caught this too) is actually approve it with a note, because as soon as it passed then you could merge. Sorry about that!